### ISSUE TITLE
Not working on new MacBook Pro with M1 processor

### ISSUE THREAD

**Issue by stevenirby** [fenneclab/hugo-bin#111]
When I install hugo-bin and try to run it, I get an expectation.

```
$(npm bin)/hugo --help
```

This is the exception I get:

```
node:events:346
      throw er; // Unhandled 'error' event
      ^

Error: spawn /Users/stevenirby/projects/randomdailyart/node_modules/hugo-bin/vendor EACCES
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
    at onErrorNT (node:internal/child_process:480:16)
    at processTicksAndRejections (node:internal/process/task_queues:81:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (node:internal/child_process:288:12)
    at onErrorNT (node:internal/child_process:480:16)
    at processTicksAndRejections (node:internal/process/task_queues:81:21) {
  errno: -13,
  code: 'EACCES',
  syscall: 'spawn /Users/stevenirby/projects/randomdailyart/node_modules/hugo-bin/vendor',
  path: '/Users/stevenirby/projects/randomdailyart/node_modules/hugo-bin/vendor',
  spawnargs: [ '--help' ]
}
```

It looks like it's not downloading the correct hugo release:

`hugo_0.82.0_macOS-ARM64.tar.gz`

OR 

`hugo_extended_0.82.0_macOS-ARM64.tar.gz`

**Comment by XhmikosR** [comment]
Can you try [this branch](https://github.com/fenneclab/hugo-bin/compare/platforms)? I made this branch sometime ago but I don't have such platforms to test.

```
npm i github:fenneclab/hugo-bin#platforms
```

**Comment by stevenirby** [comment]
I pulled it down and ran the tests and it all passed.

**Comment by XhmikosR** [comment]
Should be fixed in 0.70.2.

**Comment by XhmikosR** [comment]
@stevenirby would be so kind to test the dev branch please?

```
npm i github:fenneclab/hugo-bin#dev
```
